client.go

```diff
@@ -165,9 +165,20 @@ func (client Client) NewReq(method, uri string, body io.Reader, mods ...func(*Re
 	if req.DomainName == "" {
 		req.HttpReq.URL.Path = strings.ReplaceAll(req.HttpReq.URL.Path, "{DOMAIN_UUID}", client.DomainUUID)
 	} else {
+		// Check if selected domains exists on FMC
+		if _, ok := client.Domains[req.DomainName]; !ok {
+			availableDomains := make([]string, len(client.Domains))
+			i := 0
+			for k := range client.Domains {
```

Comment:
Shouldn't this be `for _, k := range client.Domains`?

Reply:
Client Domains is `map[string]string`, where `key` is the domain name and `value`  is its UUID.
Range over this map would return `k` (domain name) and domain UUID (which I skip).
I want to have a list of domain names, hence I believe this is correct.

Reply:
Right, I thought it is a slice.